.env.dev

```diff
@@ -32,8 +31,6 @@ NEXT_PUBLIC_RNS_REGISTRY_ADDRESS=0x7d284aaac6e925aad802a53c0c69efe3764597b8
 # CR-related env variables
 NEXT_PUBLIC_BACKERS_MANAGER_ADDRESS=0x41841e316F85933247fC7b51c79B76F22eB239bc
 NEXT_PUBLIC_REWARD_DISTRIBUTOR_ADDRESS=0x83Eac3Abe6AAF4a4AAe8B067888FAD600c4cF6B3
-NEXT_PUBLIC_CYCLE_DURATION_IN_DAYS=7
-NEXT_PUBLIC_FIRST_CYCLE_START_DATE_ISO="1970-01-01T00:00:00Z"
 
 # TODO: To be removed
 NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0xc469Cc2579De5C16210e9063B4E628bF8C46bA02
```

Comment:
I think that we need to remove this as well.

Reply:
removed